.github/workflows/deploy.yml

```diff
@@ -0,0 +1,60 @@
+name: Deploy plugin
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Hi, it's a detail but is it useful ? To me it's only useful when using build matrix.

Reply:
yep you're right!

Reply:
Thanks !